code/modules/mob/living/carbon/human/species.dm

```diff
@@ -757,14 +765,24 @@ GLOBAL_LIST_EMPTY(features_by_species)
 					accessory = GLOB.tails_list_monkey[source.dna.features["tail_monkey"]]
 				if("headtails")
 					accessory = GLOB.headtails_list[source.dna.features["headtails"]]
+				if("tail_vox")
+					accessory = GLOB.tails_list_vox[source.dna.features["tail_vox"]]
+				if("vox_hair")
+					accessory = GLOB.vox_hair_list[source.dna.features["vox_hair"]]
+				if("vox_facial_hair")
+					accessory = GLOB.vox_facial_hair_list[source.dna.features["vox_facial_hair"]]
+				if("spines_vox")
+					accessory = GLOB.spines_list_vox[source.dna.features["spines_vox"]]
+				if("vox_snout")
+					accessory = GLOB.vox_snouts_list[source.dna.features["vox_snout"]]
```

Comment:
snouts, and both hairs are external organs. they do not belong here.

Reply:
removed